azure_functions_worker/dispatcher.py

```diff
@@ -310,12 +319,17 @@ def _serialize_exception(self, exc):
                     args[name] = bindings.Out()
 
             if fi.is_async:
+                logger.info('Function is async, request ID: %s,'
+                            'function ID: %s, invocation ID: %s',
```

Comment:
not sure if there is an identified for Thread Id, if yes please include in the log

Reply:
```Function is async, request ID: 03548758-0945-48d3-86e1-900a6bb42fec,function ID: 7372087d-016a-4f0f-9c81-f70c2d4c8d1b, invocation ID: 3a584e53-63d2-4087-a9e6-a1d773245bff```